docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -0,0 +1,3769 @@
+development names.
+
+For production glyph names, names that are used in shipping font files, the
+specification is set by the Postscript and Type 1 specifications, which define
+what is expected by existing PostScript interpreters. These limitations are as
+follows.
+
+A glyph name may be up to 63 characters in length, must be entirely comprised of
+characters from the following set:
+
+    ABCDEFGHIJKLMNOPQRSTUVWXYZ
+    abcdefghijklmnopqrstuvwxyz
+    0123456789 
+    .  # period
+    _  # underscore
+
+and must not start with a digit or period. The only exception is the special
+character “.notdef”.
+
+“twocents”, “a1”, and “\_” are valid glyph names. “2cents” and “.twocents” are
+“not.
+
+Development glyph names, names used in source data during the development of a
+font, have a larger supported character set. In addition to the characters
+allowed for production glyph names, the following characters must also be
+supported for development glyph names:
+
+    U+002A * Asterisk
+    U+002B + Plus sign
+    U+002D - Hyphen-minus
+    U+003A : Colon
+    U+005E ^ Circumflex accent
+    U+007C | Vertical bar
+    U+007E ~ Tilde
+
+For glyphs where the development glyph name differs from the final production
+glyph name, an implementation of the feature file syntax must be able to accept
+either name in source files, but must produce output data which contains either
+only development glyph names or only final production glyph names.
+
+A glyph name alias database may be used by the implementation of the feature
+file grammar to map from development glyph names in the source font data to
+final production glyph names. If it is used, then it is the responsibility of
+the implementation to correlate the development glyph names used in the feature
+file with the final glyph names in the font.
+
+In order disambiguate whether a text token is a range argument or a glyph name
+which contains a hyphen, the implementation should first assume that the token
+is a glyph name. If there is no glyph in the source data with a name that
+matches the token, then the implementation should interpret the token as a range
+argument.
+
+An initial backslash serves to differentiate a glyph name from an identical
+keyword in the feature file language. (See §[2.c](#2.c) for a list of keywords.)
+For example, a glyph named “table” must be specified in the feature file as:
+
+```fea
+\table
+```
+
+<a name="2.f.ii"></a>
+#### 2.f.ii. CID
+
+CIDs are represented by a non-negative `<number>` [§[2.e.i](#2.e.i)] preceded by
+a backslash. For example:
+
+    \101
+    \0
+
+<a name="2.g"></a>
+### 2.g. Glyph classes
+
+**Note:** The feature file glyph classes described in this section are not to be
+confused with glyph classes of OpenType Layout ClassDefs. The latter are
+described in the chapter [“Common Table Formats”](https://docs.microsoft.com/en-us/typography/opentype/spec/chapter2)
+in the OpenType Font File Specification.
```

Comment:
Remove line break.
Change **OpenType Font File Specification** to **OpenType Specification**. There is another of this elsewhere in the file.

Reply:
Which line break is supposed to be removed? (Github shows this in a broad context, so I’m unsure)

Reply:
the one before **in the OpenType**. See how it renders at https://github.com/adobe-type-tools/afdko/blob/5774eabba4379164dd57c9dc5c34258383e5f2a8/docs/OpenTypeFeatureFileSpecification.md#4d-lookupflag

Reply:
This cannot be resolved. I think it has to do with github’s markdown-rendering calculating the line wrap wrongly across the long link. Here is how it looks in another environment:
<img width="918" alt="Screen Shot 2019-03-21 at 08 13 56" src="https://user-images.githubusercontent.com/2049645/54739854-6ce8c700-4bb1-11e9-9e22-ec6abaec081c.png">